libs/elements/layout/buttons/src/lib/button.interface.ts

```diff
@@ -0,0 +1,11 @@
+/*Define the structure for the button data*/ 
+export interface ButtonData
+{
+    text: string
+  
+    bgColor?: string; // Background colour
+    color?: string; // text color
+    
+    hoverBgColor?: string;
+    hoverColor?: string;
+}
```

Comment:
Good schema definition but this should be stored in the model library provided in the issue spec